README.md

```diff
@@ -18,18 +17,15 @@ The fastest way to build universal application.
 
 ---
 
-:christmas_tree: **Familiar:** React compatible API with Class Component and Hooks.
-
-:candy: **Tiny:** ~6.4 KB minified + gzipped.
+💌 **Write Once, Run Anywhere:** write one codebase, run with `Web`, [`Weex`](https://weex.apache.org/), [`Node.js`](https://nodejs.org/), `Alibaba MiniApp`, `WeChat MiniProgram` and could work more container that implements [driver specification](./docs/en-US/driver-spec.md).
 
-:earth_asia: **Universal:** works with DOM, Weex, Node.js, Mini-program, WebGL and could work more container that implements [driver specification](./docs/en-US/driver-spec.md).
+⏱ **Fast:** use better performance and tiny (📦~6KB) alternative to [React](http://reactjs.org/) with the some API.
 
-:banana: **Easy:** using via `rax-cli` with zero configuration, one codebase with universal UI toolkit & APIs.
+📤 **Easy:** quick start with zero configuration, `Progressive Web App(PWA)`, `Server-Side Rendering(SSR)`, `Function as a service(FaaS)` all features can be used out of the box.
```

Comment:
quick start with zero configuration, all features like `Progressive Web App(PWA)`, `Server-Side Rendering(SSR)` and `Function as a service(FaaS)` can be used out of the box.